tests/numpy/test_nan_reduction.py

```diff
@@ -38,9 +38,9 @@ def test_nan_reductions(nps_app_inst):
     for block_shape in block_shapes:
         ba = ba.reshape(block_shape=block_shape)
         np_arr = ba.get()
-        op_params = ["nanmax", "nanmin", "nansum"]
+        op_params = ["nanmax", "nanmin", "nansum", "nanmean"]
         axis_params = [None, 0, 1]
-        # keepdims=True is a bug with some nan functions
+        # keepdims=True is buggy with some nan functions
```

Comment:
I'm a bit fuzzy on it, but when we were trying some nan stat numpy functions it wasn't working. It's probably due to us just calling reduce though, instead of doing a proper implementation. I'll remove this comment.